documentation/docs/getting-started/core-concepts.md

```diff
@@ -0,0 +1,153 @@
+
+---
+
+# Glossary
+
+Fusion.js documentation uses industry jargon that may be specific to web development (and specific technologies employed by Fusion.js) and therefore potentially not familiar to developers who are not primarily web developers. This section provides definitions for such terms:
+
+### Asset
+
+A file that is meant to be served by a web server. Examples of assets include image files, PDF files, CSS files or Javascript files. An asset file may be generated by compilers (e.g. SASS) or other tools such as image compressors, and they may be served via a CDN in a production environment.
+
+An asset is said to be static if it's served from disk without being modified.
+
+### Brotli
+
+A compression algorithm similar to gzip. Typically, it has better compression rates than gzip, but it's not supported by all browsers. Fusion.js has brotli support out of the box for the browsers that do support it.
+
+### Bundle
+
+A bundle is a Javascript file that is compiled from one or more source files and served to a browser. By default, Fusion.js compiles two bundles: a main bundle (with application logic) and a vendor bundle (with libraries). It may additionally output more bundles if an application employs code splitting.
+
+### CDN
+
+CDN stands for Content Delivery Network. It's a collection of servers around the world that are used to serve [assets](#assets) from closer physical proximity to the clients requesting them, in order to reduce network latency.
+
+### Chunk
+
+Typically chunks are an in-memory representation of a [bundle](#bundle). The term is often used when describing Webpack internals, or to refer to chunk ids (which can be thought of as bundle identifiers).
+
+### Code fence
+
+A conditional statement whose body only runs server-only code or browser-only code (i.e. `if (__NODE__) {}` and `if (__BROWSER__) {}`)
+
+### Code splitting
+
+Code splitting (aka bundle splitting) is a technique that lazily loads of parts of an application to reduce the initial download time of an application. Fusion.js supports code splitting via `fusion-react-async`.
+
+### Context
+
+A context is a state object that represents the HTTP request and response values in middlewares on the server. The context object can also hold intermediate state such as a React tree. Each HTTP request has exactly one context object associated with it, which is passed from middleware to middleware. Once all middlewares in an application have finished running for a request in the server, this state object is translated into an HTTP response. The context object can also be used by middlewares on the browser.
+
```

Comment:
Let's link to our documentation on context in this section? https://fusionjs.com/api/fusion-core#context